meta-iot2000-bsp/recipes-tools/setledcolor/setledcolor_0.1.bb

```diff
@@ -0,0 +1,17 @@
+DESCRIPTION = "This installs a small python script to handle the led color for the SIMATIC IOT2040"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${IOT2000_MIT_LICENSE};md5=838c366f69b72c5df05c96dff79b35f2"
+
+SRC_URI = "file://setledcolor.py"
+
+S = "${WORKDIR}"
+
+FILES_${PN} += "${sysconfdir}/setledcolor/setledcolor.py \
+				${bindir}/setledcolor"
+
+do_install() {
+	install -d ${D}${sysconfdir}/setledcolor
+	install -d ${D}${bindir}
+	install -m 755 ${WORKDIR}/setledcolor.py ${D}${sysconfdir}/setledcolor/
+	ln -sf /etc/setledcolor/setledcolor.py ${D}${bindir}/setledcolor
```

Comment:
I'd rather place binaries in `{,/usr}/{,s}bin` or `/usr/local/{,s}bin` instead of `/etc` according to the [FHS](http://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html). Having the binaries "available" in two places doesn't add much benefit in my opinion, possibly quite the contrary. The file ending doesn't really matter much, I would simply strip it or, if you want to hint that it's Python, keep it. Either way, I'd opt for a consistent naming of the file.